db/seeds.rb

```diff
@@ -103,7 +103,7 @@
     vegan: false,
     allergic: false,
     allergies: "",
-    status: nil,
+    status: 2,
```

Comment:
Can you use `status: :pending` or  `ApplicationLetter.statuses[:pending]`? I think this would improve readability.